adb_graphics/figures/maps.py

```diff
@@ -585,5 +595,168 @@ def _xy_mesh(self, field):
+            plt.tight_layout()
+            plt.show()
+
+        return cf
+
+    def _label_member(self):
+
+        ''' Add the member label to the top left of the plot '''
+
+        ax = self.map.ax
+        ax.text(
+            0.05, 0.90,
+            self.member,
+            fontsize=18,
+            fontweight='bold',
+            backgroundcolor="white",
+            transform=ax.transAxes,
+            )
+
+    def title(self):
+
+        ''' Draw the title for a map. '''
+
+        f = self.field
+        atime = f.date_to_str(f.anl_dt)
+        vtime = f.date_to_str(f.valid_dt)
+        ax = self.map.ax
+
+        # Analysis time (top) and forecast hour with Valid Time (bottom) on the left
+        ax.text(0.0, 0.5,
+                f"{self.model_name}: {atime}\nFcst Hr: {f.fhr}, Valid Time {vtime}",
+                alpha=None,
+                fontsize=14,
+                horizontalalignment='left',
+                verticalalignment='top',
+                transform=ax.transAxes,
+                )
+
+        level, lev_unit = f.numeric_level(index_match=False)
+        if f.vspec.get('print_units', True):
+            units = f'({f.units}, shaded)'
+        else:
+            units = f''
+
+        # Title or Atmospheric level and unit in the high center
+        if f.vspec.get('title'):
+            title = f"{f.vspec.get('title')} {units}"
+        else:
+            level = level if not isinstance(level, list) else level[0]
+            title = f'{level} {lev_unit} {f.field.long_name} {units}'
+        ax.text(0, 0.7,
+                f"{title}",
+                horizontalalignment='left',
+                verticalalignment='top',
+                fontsize=16,
+                transform=ax.transAxes,
+                )
+
+        # Two lines for hatched data (top), and contoured data (bottom) on the right
+        contoured = self._set_overlay_string()
+        ax.text(0, 0.6,
+                f"{contoured}",
+                horizontalalignment='left',
+                verticalalignment='top',
+                fontsize=14,
+                transform=ax.transAxes,
+                )
+
+
+class MapFields():
+    ''' Class that packages all the field objects need for producing
+    desired map content, i.e. an object that contains all filled
+    contours, hatched spaces, and overlayed contours needed for a full
+    product. '''
+
+    def __init__(self, main_field, fields_spec=None, map_type=None,
+                 **kwargs):
+
+        self.main_field = main_field
+        self.fields_spec = fields_spec if fields_spec is not None else {}
+        self.map_type = map_type
+        self.model = kwargs.get('model')
+        self.tile = kwargs.get('tile', 'full')
+
+    @property
+    def contours(self):
+        ''' Return the list of contour fieldData objects'''
+
+        # We won't plot contours on multipanel plots, or full global
+        # plots.
+        if self.map_type == 'enspanel':
+            return []
```

Comment:
This allows us to turn off all contours on the ensemble plots. Similarly below, we can turn them off for global plots.